src/main/resources/lang/lang_de_DE.properties

```diff
@@ -414,6 +414,7 @@ CommandFactionClaimNoFactionPermission=Deine Rolle in dieser Fraktion gibt dir n
 CommandFactionClaimMaxClaimRadius=Du darfst keinen gr��eren Radius als {0} auf einmal beanspruchen.
 CommandFactionClaimNoClaimableChunks=Hier gibt es keine beanspruchbaren Chunks.
 CommandFactionClaimReachedDemesneLimit=Du darfst derzeit nicht mehr als {0} Chunks beanspruchen.
+CommandFactionClaimNotContiguous=Dieser Anspruch ist nicht mit anderen Anspr�chen verbunden.
```

Comment:
I'm seeing a strange symbol on the line. Is that just a limitation of GitHub's UI?

Reply:
I think this is an issue with your PC Dan, since you haven't been able to commit characters with umlauts etc either.